plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -3521,16 +3576,6 @@ public void registerCoreMembers() {
          *
          * <br><b>Sample Book Script:</b><br>
          * <ol><pre>
-         * "Cosmos Book":<br>
-         *   Type: Book<br>
-         *   Title: Cosmos, a Personal Voyage<br>
-         *   Author: Carl Sagan<br>
-         *   Text:<br>
-         *   - Every one of us is, in the cosmic perspective, precious. If a human disagrees with<br>
-         *     you, let him live. In a hundred billion galaxies, you will not find another<br>
-         *   - The nitrogen in our DNA, the calcium in our teeth, the iron in our blood, the <br>
-         *     carbon in our apple pies were made in the interiors of collapsing stars. We are <br>
-         *     made of starstuff.<br>

```

Comment:
Why was this removed?

Reply:
this entire section (`*` prefixed area) needs to be merged into modern docs, not randomly have small portions removed...